Transrate `contributing/how-to-contribute` (#199)
gatsbyjs/gatsby-ja: docs/contributing/how-to-contribute.md

---
title: 貢献の方法
---

> Gatsby への貢献は、誰にとってもおもしろく、楽しく、そして教育的であると私たちは望んでいます。 Issue や新しいドキュメント、ドキュメントの更新や微調整、ブログの投稿、ワークショップなどを含む、あらゆる貢献を歓迎します。

Author: `new docs as well as updates and tweaks`の部分がちょっと不安です。

> Gatsby への貢献は、誰にとってもおもしろく、楽しく、そして教育的であると私たちは望んでいます。 Issue や新しいドキュメント、ドキュメントの更新や微調整、ブログの投稿、ワークショップなどを含む、あらゆる貢献を歓迎します。

Reviewer: ```suggestion
私たちは Gatsby への貢献を、誰にとってもおもしろく、楽しく、そして教育的であることを望んでいます。Issue や新しいドキュメントの作成、ドキュメントの更新や微調整、ブログの投稿、ワークショップなどを含む、あらゆる貢献を歓迎します。
```

## どのように貢献を始めたらいいか分からない？

> 心配な場合やどこから始めたらいいか分からない場合は、[どこに参加するか](/contributing/where-to-participate/)のページを確認しましょう。 Twitter では [Marcy Sutton (@marcysutton)](https://twitter.com/marcysutton) や [@GatsbyJS](https://twitter.com/gatsbyjs) 、 [Discord](https://gatsby.dev/discord) では Gatsby チームの誰かに質問できます。また、 [Issue の提出](/contributing/how-to-file-an-issue/)や、メンテナーによる指導を得ることができます。

Reviewer: 原文が Issue を提出→メンテナーがそれに対して Guidance を行うと言う内容なので、少し変更しました。
指導だと少し硬い感じがするかなと思ってガイドにしましたがそこはimoです。
```suggestion
心配な場合やどこから始めたらいいか分からない場合は、[どこに参加するか](/contributing/where-to-participate/)のページを確認しましょう。Twitter では [Marcy Sutton (@marcysutton)](https://twitter.com/marcysutton) や [@GatsbyJS](https://twitter.com/gatsbyjs) 、[Discord](https://gatsby.dev/discord) では Gatsby チームの誰かに質問できます。また、 [Issue を提出](/contributing/how-to-file-an-issue/)することで、メンテナーがガイドしてくれるでしょう。
```

> 心配な場合やどこから始めたらいいか分からない場合は、[どこに参加するか](/contributing/where-to-participate/)のページを確認しましょう。Twitter では [Marcy Sutton (@marcysutton)](https://twitter.com/marcysutton) や [@GatsbyJS](https://twitter.com/gatsbyjs) 、[Discord](https://gatsby.dev/discord) では Gatsby チームの誰かに質問できます。また、 [Issue を提出](/contributing/how-to-file-an-issue/)することで、メンテナーがガイドしてくれるでしょう。

Reviewer 2: ```suggestion
心配な場合やどこから始めたらいいか分からない場合は、[どこに参加するか](/contributing/where-to-participate/)のページを確認しましょう。Twitter では [Marcy Sutton (@marcysutton)](https://twitter.com/marcysutton) や [@GatsbyJS](https://twitter.com/gatsbyjs)、[Discord](https://gatsby.dev/discord) では Gatsby チームの誰かに質問できます。また、[Issue を提出](/contributing/how-to-file-an-issue/)することで、メンテナーがガイドしてくれるでしょう。
```

## ペアプログラミング

> 一緒に取り組みたいことがあれば、 Gatsby はコミュニティーに無料の[ペアプログラミングセッション](/contributing/pair-programming/)を提供しています。貢献についての質問やペアリングのアイデアがある場合は、お問い合わせください！

Author: `pair on`が日本語だとなんて訳すかわかりませんでした。

> 一緒に取り組みたいことがあれば、 Gatsby はコミュニティーに無料の[ペアプログラミングセッション](/contributing/pair-programming/)を提供しています。貢献についての質問やペアリングのアイデアがある場合は、お問い合わせください！

Reviewer: `pair on` はこのコンテキストでは「ペアプロする」ですね。ペアプロしたい題材があればお問い合わせください！的な内容で良いと思います。
```suggestion
一緒に取り組みたいことがあれば、Gatsby はコミュニティーに無料の[ペアプログラミングセッション](/contributing/pair-programming/)を提供しています。貢献についての質問やペアプログラミングで取り組みたい題材がある場合は、お問い合わせください！
```

Author: なるほど！コンテキストによるんですね。
